src/sentry/web/client_config.py

```diff
@@ -186,6 +186,9 @@ def get_client_config(request=None):
         "demoMode": settings.DEMO_MODE,
         "enableAnalytics": settings.ENABLE_ANALYTICS,
         "validateSUForm": getattr(settings, "VALIDATE_SUPERUSER_ACCESS_CATEGORY_AND_REASON", False),
+        # TODO: to be implemented in the future when customer domain work evolves
+        "apiUrl": None,
+        "organizationUrl": None,
```

Comment:
We could default these to the current application host? That should let us start using these values with no effect on the application's behavior.

Reply:
@markstory I've used `options.get("system.url-prefix")` as the default value for these fields.